code/src/hardware_usart.cpp

```diff
@@ -17,10 +16,9 @@ namespace r2d2 {
         return hardware_usart->US_RHR;
     }
 
-    hardware_usart_c::hardware_usart_c(unsigned int baudrate, uart_ports_c usart_port) :
-        baudrate(baudrate),
-        usart_port(usart_port)
-    {
+    hardware_usart_c::hardware_usart_c(unsigned int baudrate,
+                                       uart_ports_c usart_port)
+        : baudrate(baudrate), usart_port(usart_port) {
```

Comment:
Why do the parameters each get their own line, but the initializer list arguments do not?

Reply:
that's how the .clang-format wants it